A/armadillo/build_tarballs.jl

```diff
@@ -20,31 +20,11 @@ FLAGS=(-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}
        -DCMAKE_BUILD_TYPE=Release
        -DBUILD_SHARED_LIBS=ON)
 
-if [[ "${nbits}" == 64 ]] && [[ "${target}" != aarch64* ]]; then
-    FLAGS+=(-Dopenblas_LIBRARY="${libdir}/libopenblas64_.${dlext}")
-    # Force Armadillo's CMake configuration to accept OpenBLAS as a LAPACK
-    # replacement.
-    FLAGS+=(-DLAPACK_LIBRARY="${libdir}/libopenblas64_.${dlext}")
+if [[ "${nbits}" == 64 ]]; then
 
-    SYMB_DEFS=()
-    for sym in sasum dasum snrm2 dnrm2 sdot ddot sgemv dgemv cgemv zgemv sgemm dgemm cgemm zgemm ssyrk dsyrk cherk zherk; do
-        SYMB_DEFS+=("-D${sym}=${sym}_64")
-    done
     if [[ "${target}" == *-apple-* ]] || [[ "${target}" == *-mingw* ]]; then
-        FLAGS+=(-DALLOW_OPENBLAS_MACOS=ON)
+        FLAGS+=(-DALLOW_BLAS_LAPACK_MACOS=ON)
     fi
-
-    for sym in cgbcon cgbsv cgbsvx cgbtrf cgbtrs cgecon cgees cgeev cgeevx cgehrd cgels cgelsd cgemm cgemv cgeqrf cgesdd cgesv cgesvd cgesvx cgetrf cgetri cgetrs cgges cggev cgtsv cgtsvx cheev cheevd cherk clangb clange clanhe clansy cpbtrf cpocon cposv cposvx cpotrf cpotri cpotrs ctrcon ctrsyl ctrtri ctrtrs cungqr dasum ddot dgbcon dgbsv dgbsvx dgbtrf dgbtrs dgecon dgees dgeev dgeevx dgehrd dgels dgelsd dgemm dgemv dgeqrf dgesdd dgesv dgesvd dgesvx dgetrf dgetri dgetrs dgges dggev dgtsv dgtsvx dlahqr dlangb dlange dlansy dlarnv dnrm2 dorgqr dpbtrf dpocon dposv dposvx dpotrf dpotri dpotrs dstedc dsyev dsyevd dsyrk dtrcon dtrevc dtrsyl dtrtri dtrtrs ilaenv sasum sdot sgbcon sgbsv sgbsvx sgbtrf sgbtrs sgecon sgees sgeev sgeevx sgehrd sgels sgelsd sgemm sgemv sgeqrf sgesdd sgesv sgesvd sgesvx sgetrf sgetri sgetrs sgges sggev sgtsv sgtsvx slahqr slangb slange slansy slarnv snrm2 sorgqr spbtrf spocon sposv sposvx spotrf spotri spotrs sstedc ssyev ssyevd ssyrk strcon strevc strsyl strtri strtrs zgbcon zgbsv zgbsvx zgbtrf zgbtrs zgecon zgees zgeev zgeevx zgehrd zgels zgelsd zgemm zgemv zgeqrf zgesdd zgesv zgesvd zgesvx zgetrf zgetri zgetrs zgges zggev zgtsv zgtsvx zheev zheevd zherk zlangb zlange zlanhe zlansy zpbtrf zpocon zposv zposvx zpotrf zpotri zpotrs ztrcon ztrsyl ztrtri ztrtrs zungqr; do
-        SYMB_DEFS+=("-D${sym}=${sym}_64")
-    done
-
-    export CXXFLAGS="${SYMB_DEFS[@]}"
-
-    # Force the configuration parameter ARMA_BLAS_LONG to be true, as in our
-    # setting 64-bit systems are going to need a 64-bit integer to be used for
-    # Armadillo's `blas_int` type.
-    sed -i 's|// #define ARMA_BLAS_LONG$|#define ARMA_BLAS_LONG|' ../include/armadillo_bits/config.hpp.cmake
-    cat ../include/armadillo_bits/config.hpp.cmake
```

Comment:
I'm doubtful that this should be removed.  Armadillo is essentially a header-only C++ wrapper around LAPACK and BLAS, and it has to reference the correct symbols from OpenBLAS.  Unless Yggdrasil has changed how it distributes OpenBLAS on OS X, I strongly suspect that removing this is incorrect.